### ISSUE TITLE
The markup declarations contained or pointed to by the document type declaration must be well-formed

### ISSUE THREAD

**Issue by tresf** [qzind/tray#809]
Locating Firefox can fail with the following:

```
Fatal Error] PropertyList-1.0.dtd:1:2: The markup declarations contained or pointed to by the document type declaration must be well-formed.
[WARN] 2021-05-07 11:26:07,155 @ qz.installer.certificate.firefox.locator.MacAppLocator:64
	Could not retrieve app listing for FIREFOX
org.xml.sax.SAXParseExceptionpublicId: -//Apple//DTD PLIST 1.0//EN; systemId: http://www.apple.com/DTDs/PropertyList-1.0.dtd; lineNumber: 1; columnNumber: 2; The markup declarations contained or pointed to by the document type declaration must be well-formed.
	at java.xml/com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(DOMParser.java:261)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderImpl.parse(DocumentBuilderImpl.java:339)
	at java.xml/javax.xml.parsers.DocumentBuilder.parse(DocumentBuilder.java:122)
	at qz.installer.certificate.firefox.locator.MacAppLocator.locate(MacAppLocator.java:62)
	at qz.installer.certificate.firefox.FirefoxCertificateInstaller.install(FirefoxCertificateInstaller.java:81)
	at qz.installer.Installer.certGen(Installer.java:208)
	at qz.ws.PrintSocketServer.main(PrintSocketServer.java:81)
```

**Comment by tresf** [comment]
This is a result of parsing the output of `system_profiler SPApplicationsDataType -xml` using `DocumentBuilderFactory.newInstance().newDocumentBuilder().parse(...)`

**Comment by tresf** [comment]
Turns out the XML parser is a bit strict, so if the DTD file reference in the `<!DOCTYPE>` tag is not available the whole thing breaks.  Fixed via c04b510 by ignoring the DTD altogether, which seems to fix this edge-case.  This fix will be available starting with QZ Tray 2.1.3.
